client/src/components/jwt.tsx

```diff
@@ -0,0 +1,28 @@
+import UserContext from '../context/UserContext';
+import { 
+	buildRequestParams 
+} from './RequestBuilder';
+
+export const validate = async (user:any) => {
+	const params = buildRequestParams({
+		token: user.userData.token,
+	});
+
+	const request = await fetch(`api/validate?${params}`, {
+		method: 'POST'
+	});
+
+	const response = await request.json();
+	console.log(response);
+      
+	if(response == true) {
```

Comment:
een cleaner manier is om de errors vroeger te catchen dan na all de checks: 
`if (!response) {
setUser({})
return false;
}
return true;
`